### PR TITLE
Add note about WSI origin location

### LINKED ISSUE #1191: Unspecified WSI coordinate system origin?
I cannot find in the specification how is a `VkImage` interpreted by the presentation engine. I.e. in what corner of the surface\window\display would a texel (0,0) show up.

Is this trully unspecified and implementation dependent (in which case I think there should be a Note)? Or is this an omission?

### PR DESCRIPTION
**Pull request by krOoze**
fix #1191

@stonesthrow pls review

### CODE REVIEW

**Comment by stonesthrow** [comment]
Thank you for the PR. We'll review this week.

File: chapters/VK_KHR_swapchain/wsi.txt [hunk 1]
```diff
@@ -1166,6 +1166,18 @@ the presentation engine.
...
+====
+The origin of the native orientation of the surface coordinate system is not
+specified in the Vulkan specification; it depends on the platform.
+For most platforms the origin is by default upper-left, meaning the pixel
+of the presented slink:VkImage at coordinates [eq]#(0,0)# would appear at
+the upper left pixel of the platform surface (assuming
+ename:VK_SURFACE_TRANSFORM_IDENTITY_BIT_KHR, and the display standing the
+right way up).
+====
+
```
Comment (cubanismo) [inline chapters/VK_KHR_swapchain/wsi.txt]: The note feels a little arbitrary sitting here among queue operation scope language and whatnot, but there doesn't seem to be any great place for this note to live, so I guess this is as good as any.

Comment (krOoze) [reply chapters/VK_KHR_swapchain/wsi.txt]: The note is below `vkQueuePresentKHR`, where it feels most useful to me.

Yea, I have been strugling to find the best place. There still kinda feels like there is some language missing describing the `vkQueuePresent` behavior in general, which would be my first choice to put the Note there.

The previous sentence at least talks about the swapchain images, which felt like the least non-relevant place to put the Note in.

**Comment by stonesthrow** [comment]
Vulkan Si agreed to PR proposal - approved. thx KrOose.

**Pull request merged**